library/objective-c/EnvoyFilterCallbacksImpl.m

```diff
@@ -0,0 +1,20 @@
+#import "library/objective-c/EnvoyEngine.h"
+
+#pragma mark - EnvoyHTTPFilterCallbacksImpl
+
+@implementation EnvoyHTTPFilterCallbacks {
+}
+
+- (instancetype)init {
+  self = [super init];
+  if (!self) {
+    return nil
+  }
+  return self;
+}
```

Comment:
Why do we need this?

Reply:
The next PR has stuff that needs to go here, so I went ahead and dropped in the boilerplate when I created the class.